README.md

```diff
@@ -565,6 +575,35 @@ In order to add the hastexo Xblock through Studio, open the unit where you want
 it to go.  Add a new component, select `Advanced`, then select the `Lab`
 component.  This adds the XBlock.  Edit the Settings as explained above.
 
+### Using the hastexo XBlock in a content library
+
+This XBlock is usable in [content libraries](https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/course_components/libraries.html).
+It supports adding lab instructions as child blocks, so that when the block is
+randomized, the instructions are bundled together with it.
+
+To add the XBlock to the library via Studio, make sure it is configured as one
+of the `ADVANCED_PROBLEM_TYPES` in `cms.env.json`, then select it as such when
+adding content to your library.  (Note: as of Open edX Ironwood, the ability to
+do so requires running a patched version of `edx-platform`.)
```

Comment:
Can we have a link to the patched branch here, please?

Reply:
Done.